src/utils/EmailType.tsx

```diff
@@ -0,0 +1,34 @@
+import { GenericIdentifierType } from './GenericIdentifierType';
+import { FunctionalComponent, h } from '@stencil/core';
+
+/**
+ * This class specifies a custom renderer for Email addresses.
+ * @extends GenericIdentifierType
+ */
+export class EmailType extends GenericIdentifierType {
+  getSettingsKey(): string {
+    return 'DateType';
```

Comment:
This should be the same name as your Renderer. In your case, this would be: `EmailType`

Reply:
This is also `DateType` for the class `URLType`, is this unintentional?

Reply:
Yep, I forgot to change that. If you are already at it, you can do it...